src/Microsoft.DotNet.Tools.Test/Microsoft.DotNet.Tools.Test.xproj

```diff
@@ -5,17 +5,16 @@
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
 
-  <Import Project="$(VSToolsPath)\DotNet\Microsoft.DotNet.Props" Condition="'$(VSToolsPath)' != ''"  />
+  <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" Condition="'$(VSToolsPath)' != ''" />
```

Comment:
The intention is to match all the other xproj's in the repo.  I spot checked a few of them, and I only saw DNX being used, so that's what I used here.

I opened https://github.com/dotnet/cli/issues/4156 for fixing this everywhere.